### DISCUSSION TITLE
3.0.0 Release

### DISCUSSION THREAD

**Discussion by ssiva** [gklijs/schema_registry_converter#83] [General]
Hi @gklijs ,

Appreciate all your work/effort. Is there any plan for the 3.0.0 release with the apache-avro crate ?

Thanks,
Siva

**Comment by eblackw2** [comment]
@ssiva, @gklijs, was there any word on when 3.0.0 was going to be released? I have a project that has to use apache-avro because it fixed a bug that was affecting deserialization

**Reply by gklijs** [reply]
When it's done. most remaining issues are low effort, except for https://github.com/gklijs/schema_registry_converter/issues/79, which I really want included to stay compatible. I have some time next weekend, but no promises. As that issue won't change the API I might release 3.0.0 without, although I don't like to do so.

**Comment by gklijs** [comment]
The integration test for the common types succeeded. So know it's down to some unit tests to close that issue. And wrapping up the other open issues. I hope to release it soon, as in within a week from now.

**Comment by ssiva** [comment]
Thank you @gklijs

**Comment by gklijs** [comment]
Just released the crate. As the other open issues don't need to break the API, they could be done later, when needed.

**Comment by ultrabug** [comment]
Congratulations and thanks @gklijs !

**Comment by eblackw2** [comment]
Thanks for the quick turn around @gklijs !
